### PR TITLE
Create initial ustreamer manpage

### PR DESCRIPTION
**Pull request by reedy**
Fixes https://github.com/pikvm/ustreamer/issues/62

I don't think this is perfect, there needs to be some more tweaking, and probably a few more additions/updates in a few places.. But should be good for pass 1, and allows someone else to review/CR it

And it doesn't include the GPIO options

Also would need plumbing into the packaging so it's installed on various OS too

### CODE REVIEW

**Comment by mdevaev** [comment]
Plz add manpage's version to bumpversion.cfg. Otherwise, I will constantly forget to update it. It might also make sense to add this to the description: https://github.com/pikvm/ustreamer#usage

**Comment by reedy** [comment]
> It might also make sense to add this to the description: https://github.com/pikvm/ustreamer#usage

I'll see about adding a version of that when I can work out the formatting

File: .bumpversion.cfg [hunk 1]
```diff
@@ -17,3 +17,7 @@ replace = pkgver={new_version}
 [bumpversion:file:pkg/openwrt/Makefile]
 search = PKG_VERSION:={current_version}
 replace = PKG_VERSION:={new_version}
+
+[bumpversion:file:ustreamer.1]
+search = version "{current_version}"
```
Comment (mdevaev) [inline .bumpversion.cfg:22]: Mus be `"version {current_version}"` I guess. Same to replace.

**Comment by reedy** [comment]
Nearly there...

Screenshot 2020-12-01 at 22 39 41

Just need to remove some of those newlines

**Comment by reedy** [comment]
Screenshot 2020-12-01 at 22 46 16

File: .bumpversion.cfg (hunk 1 above)
Comment (reedy) [reply .bumpversion.cfg]: Yup, that looks to work

```
diff --git a/ustreamer.1 b/ustreamer.1
index fcd7b51..e86d4d2 100644
--- a/ustreamer.1
+++ b/ustreamer.1
@@ -1,6 +1,6 @@
 .\" Manpage for ustreamer.
 .\" Open an issue or pull request to https://github.com/pikvm/ustreamer to correct errors or typos
-.TH USTREAMER 1 "version 2.2" "November 2020"
+.TH USTREAMER 1 "version 2.3" "November 2020"
```

**Comment by mdevaev** [comment]
Great work!

**Pull request merged**